sites/partners/cypress/integration/06-user-mangement.spec.ts

```diff
@@ -0,0 +1,47 @@
+describe("User Mangement Tests", () => {
+  it("as admin user, should show all users regadless of jurisdiction", () => {
+    cy.login()
+    cy.visit("/")
+    cy.getByTestId("Users-1").click()
+    const rolesArray = [
+      "Partner",
+      "Partner",
+      "Administrator",
+      "Administrator",
+      "Jurisdictional Admin",
+    ]
+    cy.get(`.ag-center-cols-container [row-id="0"] [col-id="roles"]`)
```

Comment:
No worries if this is challenging to update, but this selector is very tied to implementation details - is there a way to use a more generic selector or a test id?

Reply:
I think we are somewhat limited as we are using the `ag-grid-react` package, I couldn't find a better way to target rows/column specific data